apps/condo/domains/ticket/components/TicketForm/UpdateTicketForm.tsx

```diff
@@ -195,10 +249,16 @@ export const UpdateTicketForm: React.FC<IUpdateTicketForm> = ({ id }) => {
             result['invoices'] = invoices.map(invoice => invoice.id)
         }
 
+        if (observers.length > 0) {
```

Comment:
please double check the types here, if observers is not an array – page will fail